internal/helpers/messages/error.go

```diff
@@ -21,7 +21,7 @@ const (
 	// USED IN USE CASES: Fired when the project path is invalid
 	MsgErrorProjectPathNotValid = "project path is invalid: "
 	// USED IN USE CASES: Fired when an path of json is not valid in configs
-	MsgErrorJSONOutputFilePathNotValid = "JSON File path is required or is invalid: "
```

Comment:
Since we change the erro the message I think that we should we rename the const to `MsgErrorOutputFilePathNotValid`

Reply:
Yes me too, however the name refers to the name of the flag, which is called JSONOutputFilePath.